src/makeCalculator.js

```diff
@@ -38,6 +38,42 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    result: 0,
+
+    add: function(num) {
```

Comment:
```suggestion
    add(num) {
```
![image](https://github.com/mate-academy/js_advanced_calculator/assets/96409563/13be28df-29e2-4190-ac56-10bd109b3b43)